Framework/CurveFitting/inc/MantidCurveFitting/FuncMinimizers/FABADAMinimizer.h

```diff
@@ -62,44 +62,70 @@ class DLLExport FABADAMinimizer : public API::IFuncMinimizer {
   /// Finalize minimization, eg store additional outputs
   void finalize() override;
 
-private:
-  /// Returns the step from a Gaussian given sigma = Jump
-  double GaussianStep(const double &Jump);
+  /// Public methods only for testing purposes
+
   /// If the new point is out of its bounds, it is changed to fit in the bound
   /// limits
-  void BoundApplication(const size_t &ParameterIndex, double &new_value,
+  void boundApplication(const size_t &parameterIndex, double &newValue,
                         double &step);
+
+private:
+  /// Returns the step from a Gaussian given sigma = Jump
+  double gaussianStep(const double &jump);
   /// Applied to the other parameters first and sequentially, finally to the
   /// current one
-  void TieApplication(const size_t &ParameterIndex, GSLVector &new_parameters,
-                      double &new_value);
+  void tieApplication(const size_t &parameterIndex, GSLVector &newParameters,
+                      double &newValue);
   /// Given the new chi2, next position is calculated and updated.
   /// m_changes[ParameterIndex] updated too
-  void AlgorithmDisplacement(const size_t &ParameterIndex,
-                             const double &chi2_new, GSLVector &new_parameters);
+  void algorithmDisplacement(const size_t &parameterIndex,
+                             const double &chi2New, GSLVector &newParameters);
   /// Updates the ParameterIndex-th parameter jump if appropriate
-  void JumpUpdate(const size_t &ParameterIndex);
+  void jumpUpdate(const size_t &parameterIndex);
   /// Check for convergence (including Overexploration convergence), updates
   /// m_converged
-  void ConvergenceCheck();
+  void convergenceCheck();
   /// Refrigerates the system if appropriate
-  void SimAnnealingRefrigeration();
+  void simAnnealingRefrigeration();
   /// Decides wheather iteration must continue or not
-  bool IterationContinuation();
+  bool iterationContinuation();
+  /// Output Markov chains
+  void outputChains();
+  /// Output converged chains
+  void outputConvergedChains(size_t convLength, int nSteps);
+  /// Output cost function
+  void outputCostFunctionTable(size_t convLength, double mostProbableChi2);
+  /// Output PDF
+  double outputPDF(size_t convLength,
+                   std::vector<std::vector<double>> &reducedChain);
+  /// Output parameter table
+  void outputParameterTable(const std::vector<double> &bestParameters,
+                            const std::vector<double> &errorsLeft,
```

Comment:
They are an old piece of code, where, instead of calculating the right and left variance of the pdf, it took the percentiles 32 and 68 to get the error (it came from the percentiles associated with the Gaussian distribution)
It should be changed